java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/cp/IAtomicLongSharedCounterTest.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2008-2023, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hazelcast.simulator.tests.cp;
+
+import com.hazelcast.cp.IAtomicLong;
+import com.hazelcast.simulator.hz.HazelcastTest;
+import com.hazelcast.simulator.test.BaseThreadState;
+import com.hazelcast.simulator.test.annotations.AfterRun;
+import com.hazelcast.simulator.test.annotations.Setup;
+import com.hazelcast.simulator.test.annotations.Teardown;
+import com.hazelcast.simulator.test.annotations.TimeStep;
+import com.hazelcast.simulator.test.annotations.Verify;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Shared counter ({@link IAtomicLong}) that is incremented/read by the configured number of clients/threads.
+ */
+public class IAtomicLongSharedCounterTest extends HazelcastTest {
+    private IAtomicLong sharedCounter;
```

Comment:
I think there should be a configurable number of IAtomicLongs. Everyone hammering on the same AtomicLong is not always that interesting.

Reply:
There's already a test for this, I believe `IAtomicLongTest`. Perhaps mine is redundant?

Reply:
Perhaps. Simplification is always good and Simulator and we don't want to add more superfluous tests to simulator.

Reply:
Sure, I agree -- I'll remove the test I added.